src/infrastructures/Config.ts

```diff
@@ -4,23 +4,22 @@ import {
   ConfigType,
   EngineId,
   configSchema,
+  DefaultStyleId,
 } from "@/type/preload";
 
-const migrations: [string, (store: Config) => void][] = [
+const migrations: [string, (store: Record<string, unknown>) => unknown][] = [
   [
     ">=0.13",
     (store) => {
       // acceptTems -> acceptTerms
       const prevIdentifier = "acceptTems";
-      // @ts-expect-error 削除されたパラメータ。
-      const prevValue = store.get(prevIdentifier, undefined) as
-        | AcceptTermsStatus
-        | undefined;
+      const prevValue = store.prevIdentifier as AcceptTermsStatus | undefined;
```

Comment:
ここって`store.prevIdentifier`でいいんでしたっけ。 👀 
```suggestion
      const prevValue = store["prevIdentifier"] as AcceptTermsStatus | undefined;
```
こうかも？